README.rst

```diff
@@ -90,8 +90,69 @@ testing with live email addresses.  It works like this:
     client.destroy_account(acct.email, "MySecretPassword")
 
 
+Passing tokens and assertions to other applications
+===================================================
+
+PyFxA provides a ``fxa-client`` that you can use to export Bearer
+Tokens and Browser ID assertions.
+
+
+Get a Bearer Token for an existing account
+------------------------------------------
+
+.. code-block:: bash
+
+    fxa-client --bearer --auth you@domain.tld \
```

Comment:
Stylistically, I think these would work better as "sub-commands" rather than options (like e.g. argparse's add_subparsers() functionality).  Something like:

```
fxa-client token --auth [...etc...]
```

And

```
fxa-client assertion --auth [...etc...]
```